custom/ilsgateway/zipline/filters.py

```diff
@@ -34,3 +37,22 @@ def options(self):
              EmergencyOrderStatusUpdate.STATUS_CANCELLED_IN_FLIGHT),
 
         ]
+
+
+class OrderIdChoiceFilter(BaseMultipleOptionFilter):
+
+    slug = 'orders_id'
+    label = 'Orders id'
+
+    is_paginated = True
+
+    @property
+    def pagination_source(self):
+        return reverse_lazy('zipline_orders_view', kwargs={'domain': self.domain})
+
+    @property
+    def options(self):
+        return [
+            {'id': pk, 'text': pk}
+            for pk in EmergencyOrder.objects.filter(domain=self.domain).values_list('pk', flat=True)
```

Comment:
this list can end up getting very large. i think for now it would be best to remove the ZiplineOrdersView and only allow filtering on one order_id, which we take from a text input